js/master.js

```diff
@@ -23,7 +24,13 @@ function getClosestFactors(num) {
     }
 }
 
-window.onload = function () {
+window.onload = function() {
+    if (navigator.userAgent.includes('Firefox')) {
+        document.getElementById('favicon').setAttribute('href', 'chrome://branding/content/icon32.png')
+    }else {
+        document.getElementById('favicon').setAttribute('href', '/favicon.svg')
+    }
```

Comment:
this could be simplified to a `?:`, but I'll allow it

Reply:
readability